Gemfile

```diff
@@ -20,7 +20,7 @@ gem 'rails', '~> 6.0'
 gem 'mail', '~> 2.7.1'
 
 # Needed for XML serialization of ActiveRecord::Base
-gem "activejob-uniqueness", github: "3scale/activejob-uniqueness", branch: "main"
+gem "activejob-uniqueness"
```

Comment:
Can you please also swap it with the next line?
The `# Needed for XML serialization of ActiveRecord::Base` comment actually belongs to `'activemodel-serializers-xml'`, so having it here is confusing :grimacing:

Reply:
Done!